.github/workflows/ci-rsc.yml

```
@@ -0,0 +1,61 @@
+name: ci-rsc
+"on":
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "packages/plugin-rsc/**"
+      - "pnpm-lock.yaml"
+      - ".github/workflows/ci-rsc.yml"
+
+concurrency:
+  group: ci-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+      - run: corepack enable
```

```suggestion
      - uses: pnpm/action-setup@a7487c7e89a18df4991f7f222e4898a00d66ddda # v4.1.0
```
Just to align with other workflows.